packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -3379,7 +3380,11 @@ export interface IDetailsListProps extends IBaseProps<IDetailsList>, IWithViewpo
     constrainMode?: ConstrainMode;
     disableSelectionZone?: boolean;
     dragDropEvents?: IDragDropEvents;
+    // @deprecated
+    enableShimmer?: boolean;

```

Comment:
Another new prop that's being deprecated immediately? I can't find where this is being picked up from, either. I don't see a corresponding change in `DetailsList.types.ts`.

Reply:
Result of an incorrect merge with `6.0`

Reply:
I should manually removed it since enableShimmer is not my change

Reply:
You can't manually edit this file, it will just get regenerated based on PR contents. This file should **only** be updated via `npm run update-api`.